src/main/kotlin/frc/robot/auto/network_tables/NetworkTableIO.kt

```diff
@@ -0,0 +1,15 @@
+package frc.robot.auto.network_tables
+
+import org.team9432.annotation.Logged
+
+interface NetworkTableIO {
+    var inputs: LoggedNetworkTableInputs
+
+    fun updateInputs() {}
+
+    @Logged
+    open class NetworkTableInputs {
+        var targetBranchPose: Int = 0
+        var targetReefPose: Int = 0
```

Comment:
If this includes the coral station location the name should change. Why not make two topics though? Now if the driver selected a coral station last the robot code will have no way the know the desired reef pose